storybook/main.ts

```diff
@@ -42,11 +49,12 @@ const config = {
       propFilter: (prop): boolean => {
         const isHTMLElementProp =
           prop.parent?.fileName.includes("node_modules/@types/react") ?? false
-
         return !isHTMLElementProp
       },
     },
   },
+  docs: {
+    autodocs: true,
+  },
```

Comment:
We don't want this :)

Reply:
I wonder if we should be explicit and put `autodocs: "tag"` (or whatever it should be) in to stop the auto migrate prompt next time 🤔

Reply:
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#autodocs-changes

>However, as of 7.0, the new default is 'tag'

Reply:
Ok, I'm then just going to remove the setting, as it will default to `tag`.